README.md

```diff
@@ -1,16 +1,17 @@
-# Web Monetization Tools
+# Publisher Tools
 
-<a href="#what-is-web-monetization-tools">
-  <img src="https://github.com/interledger/web-monetization-tools/blob/25fff6ab48b052ac1190cf3734cb96aba99ed9a2/docs/preview.png?raw=true" alt="Web monetization tools preview image">
+<a href="#what-is-the-publisher-tools">
```

Comment:
Should this be #what-are-the-publisher-tools ?

Reply:
Absolutely :)